prac_06/guitar.py

```diff
@@ -0,0 +1,16 @@
+class Guitar:
+    def __init__(self, name="", year=0, cost=0.0, age=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+        self.age = age
+
+    def __str__(self):
+        return "{} ({}) : $ {}".format(self.name, self.year, self.cost)
+
+    def get_age(self, current_year=2022):
```

Comment:
you can just do:
return 2022 - self.year

Reply:
That looks better :)
..changed code to

def get_age(self):
        return 2022 - self.year